rdr_service/api/genomic_cloud_tasks_api.py

```diff
@@ -256,11 +256,14 @@ class IngestDataFilesTaskApi(BaseGenomicTaskApi):
     """
     def post(self):
         super(IngestDataFilesTaskApi, self).post()
+        logging.info(f'self data: {self.data}')
```

Comment:
Do we need two logs here?

Reply:
No, this was for debugging. I will remove.